backend/src/services/interview/interview.service.ts

```diff
@@ -143,18 +145,34 @@ class Interview {
 
   public async update(data: {
     id: number;
+    user: UserWithPermissions;
     interviewUpdateInfoRequestDto: InterviewsUpdateRequestDto;
   }): Promise<InterviewsByIdResponseDto> {
-    const { id, interviewUpdateInfoRequestDto } = data;
+    const { id, interviewUpdateInfoRequestDto, user } = data;
     const { interviewerUserId, status, interviewDate } =
       interviewUpdateInfoRequestDto;
 
-    const interview = await this.#interviewRepository.update({
-      id,
-      interviewerUserId,
-      status,
-      interviewDate,
+    const hasInterviewsPermission = checkHasPermission({
+      permissionKeys: [PermissionKey.MANAGE_INTERVIEWS],
+      userPermissions: user.permissions,
     });
+    const hasInterviewerUserId = interviewerUserId !== undefined;
+
+    if (!hasInterviewsPermission && hasInterviewerUserId) {
+      throw new PermissionsError({
+        message: ExceptionMessage.PERMISSION_LACK_FOR_INTERVIEWER_UPDATE,
+        status: HttpCode.FORBIDDEN,
+      });
+    }
+
+    const interviewUpdateInfoDependOnPermission = !hasInterviewsPermission
+      ? { status, interviewDate }
+      : interviewUpdateInfoRequestDto;
```

Comment:
Looks like you check this above, and interviewUpdateInfoRequestDto will contain only permitted fields